### DISCUSSION TITLE
1080pm first, then 720p

### DISCUSSION THREAD

**Discussion by sendas** [pymedusa/Medusa#9163] [Q&A]
Hi

I've set Medusa to download episodes at 1080p

Sometimes there's aren't 1080p ep. available, so I would like Medusa to download 720p episodes.

Is this possible?

Thanks

**Comment by p0psicles** [comment]
Use allowed 1080, preferred 720

**Reply by sendas** [reply]
How do i do that ? 😬

**Reply by sendas** [reply]
I would prefer allowed 720p and preferes 1080p ..?

**Reply by sendas** [reply]
Found it! Thanks !

**Reply by sendas** [reply]
Now it downloads 720p and them 1080p...

I have allowed 720p and preferred 1080p ...

![image](https://user-images.githubusercontent.com/10220908/107506522-16fedd00-6b96-11eb-9dab-f955486150ac.jpeg)

**Reply by sendas** [reply]
Isn't there a way do only download 720p if 1080p is not found ? Thanks

**Reply by medariox** [reply]
Just set both in allowed qualities. Medusa will snatch the best available quality.
